filesystem/vdi.ksy

```diff
@@ -0,0 +1,228 @@
+meta:
+  id: vdi
+  endian: le
+  encoding: utf-8
+  application:
+    - VirtualBox
+    - QEMU
+    - VMWare Workstation
+  license: GPL3.0+
```

Comment:
1. Probably something like `title: VDI (native VirtualBox Disk Image)` would be helpful.
2. Suggested order of `meta` entries is somewhat different in [style guide](http://doc.kaitai.io/ksy_style_guide.html).

Reply:
It is speculated in multiple sources that VDI is an abbreviation for VirtualBox Disk Image.